src/UserManagement/UsersManager.php

```diff
@@ -409,4 +421,18 @@ static function (Throwable $e) {
             );
         }
     }
+
+    private function createDateTimeString(string $dateTimeString): string
```

Comment:
This seems to be duplicated. Maybe it would be better to extract it to separate class.

Reply:
Suggestions? I like `DateTimeStringParserWorkaroundSolutionHandler` :)

Reply:
Haha why not =)

Reply:
Called it `DateTimeStringBugWorkaround` for now, class is marked as internal, so we can rename or remove at any time.

Reply:
👍